tests/data/pandapower/pgm_input_data.json

```diff
@@ -49,10 +49,10 @@
     ],
   "asym_load":
     [
-      {"id": 16, "node": 4, "status": 1, "type": 0,"p_specified": [100000, 200000, 300000], "q_specified": [10000, 20000, 30000], "id_reference": {"table": "asymmetric_load", "index": 33}}
+      {"id": 16, "node": 4, "status": 1, "type": 0, "id_reference": {"table": "asymmetric_load", "index": 33}}
     ],
   "asym_gen":
     [
-      {"id": 17, "node": 4, "status": 1, "type": 0,"p_specified": [100000, 200000, 300000], "q_specified": [10000, 20000, 30000], "id_reference": {"table": "asymmetric_sgen", "index": 32}}
+      {"id": 17, "node": 4, "status": 1, "type": 0, "id_reference": {"table": "asymmetric_sgen", "index": 32}}
```

Comment:
Why do we skip pq_specified validation here?

Reply:
Because the validation for attributes was able to compare only single values. It needs modification to work for p_specified of asym_gen.
But thought we might solve that while making asymmetric conversions. 
I will mention this in the asymmetric conversions ticket.